SmartPlaces.Facilities/lib/IngestionManager/src/Metrics.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="Metrics.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.SmartPlaces.Facilities.IngestionManager
+{
+    internal static class Metrics
+    {
+        internal const string DefaultNamespace = "Facilities";
```

Comment:
Should this also flop back to Microsoft.SmartFacilities.Places

Reply:
Fixed